utils.js

```diff
@@ -0,0 +1,76 @@
+function generateSummary(core, metrics) {
+  body = '### Lava scan results\n\n';
+
+  if (Object.keys(metrics.vulnerability_count).length === 0) {
+    return body + `No vulnerabilities found!! 🎉🎉🎉\n\n`;
+  }
+
+  body += `| Severity | Findings |\n`;
+  body += `|---|---:|\n`;
+
+  if (metrics.vulnerability_count["success"] > 0) {
+    body += `| :white_check_mark: success | ${metrics.scan_count["success"]} |\n`;
+  }
+  if (metrics.vulnerability_count["critical"] > 0) {
+    body += `| :fire: critical | ${metrics.vulnerability_count["critical"]} |\n`;
+  }
+  if (metrics.vulnerability_count["high"] > 0) {
+    body += `| :red_circle: high | ${metrics.vulnerability_count["high"]} |\n`;
+  }
+  if (metrics.vulnerability_count["medium"] > 0) {
+    body += `| :large_orange_diamond: medium | ${metrics.vulnerability_count["medium"]} |\n`;
+  }
+  if (metrics.vulnerability_count["low"] > 0) {
+    body += `| :large_blue_diamond: low | ${metrics.vulnerability_count["low"]} |\n`;
+  }
+  if (metrics.vulnerability_count["info"] > 0) {
+    body += `| :sunny: info | ${metrics.vulnerability_count["info"]} |\n`;
+  }
+  return body;
+}
+
+async function comment(github, context, summary) {
+  const mark = `<!-- action:lava-action job:${context.job} -->`;
+  const body = summary + mark + `\n\n[Logs ${context.job}](${context.serverUrl}/${context.repo.owner}/${context.repo.repo}/actions/runs/${context.runId})`;
+
+  // Get the existing comments.
+  const { data: comments } = await github.rest.issues.listComments({
+    owner: context.repo.owner,
+    repo: context.repo.repo,
+    issue_number: context.payload.number
+  })
+
+  // Delete the comments created by the bot and containing the mark.
+  await comments
+    .filter(comment =>
+      comment.user.id === 41898282 && comment.body.includes(mark))
+    .forEach(async (comment) => {
+      try {
+        await github.rest.issues.deleteComment({
+          owner: context.repo.owner,
+          repo: context.repo.repo,
+          comment_id: comment.id
+        })
+      } catch (e) {
+        console.log(e);
+      }
+    });
+
+  await github.rest.issues.createComment({
+    owner: context.repo.owner,
+    repo: context.repo.repo,
+    issue_number: context.payload.number,
+    body: body
+  })
+}
+
+module.exports.process = async ({ github, context, core }) => {
+  const metrics = JSON.parse(require('fs').readFileSync(process.env.METRICS, 'utf8'));
+
+  var summary = generateSummary(core, metrics);
```

Comment:
`summary` can be `const`, right?

```suggestion
  const summary = generateSummary(core, metrics);
```

Reply:
done